apps/pixeloven/site/docs/recommendations/react-components.mdx

```diff
@@ -3,33 +3,31 @@ id: react-components
 title: React Components
 ---
 
-This document is general advice for creating and working with new components.
+This document is general advice for working with React components.
 
 ## Referencing source files
 
-To make referencing source files easier we have set up three simple aliases when working inside an app.
-
--   `@client` which points to `src/client/*`
--   `@server` which points to `src/server/*`
--   `@shared` which points to `src/shared/*`
-
-Example usage would be as follows:
+To make referencing source files easier we have set up a `@shared` directory alias to simplify import declarations vs having to navigate up and down the directory tree. For example, to import a Logo Atom into another component we'd write:
 
 ```javascript
 import { Logo } from "@shared/components/atoms/Logo";
 ```
 
-Alternatively we can also create an `index.ts` file in the atoms directory and export all of our **atoms** allowing us to reference multiple at once.
+Alternatively we can also create an `index.ts` file in the atoms directory and export all of our Atom components, allowing us to reference multiple at once.
+
+_Note: Be extremely cautious when importing from a shared directory such as this. If the component we're importing into is exported from the same directory as the component we're importing, it will create a circular dependency. This can create bugs that are difficult to track down, slow down our application, or even create heap memory issues that result in our application crashing._
 
 ```javascript
-import { Logo } from "@shared/components/atoms";
+import { Button, Link, Logo } from "@shared/components/atoms";
```

Comment:
I was tempted to remove this entirely since as an org we're moving away from this pattern, but instead added a note about circular dependencies below.